### DISCUSSION TITLE
Problem whith setvc

### DISCUSSION THREAD

**Discussion by Capicat** [jagrosh/MusicBot#1414] [Questions] [answered]
if i run the command (prefix)setvc/settc channelname (in<>) it tells me that a channel whith that name dosn't exsist

**Comment by YouKaih** [comment]
Hi, can you screenshot it? It works for me just fine

![obrázek_2024-01-16_132459682](https://github.com/jagrosh/MusicBot/assets/57845237/3ca70809-68e1-43e8-9eb3-53aaa0b6ae93)

**Reply by Capicat** [reply, marked as answer]
sorry i forgot about this post. i ended up finding out what the problem was.. a spelling issue or something like that. thanks for replying though

**Reply by YouKaih** [reply]
No problem :)
